### PR TITLE
Feature/core 639 sprint feedback

### PR DESCRIPTION
**Pull request by Tschuck**

### CODE REVIEW

File: dapps/ui.libs/src/bccUtils.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      bcc.lodash,
+      accountDetails || await profile.getProfileProperty('accountDetails') || { }
+    );
+
+    // for companies load directly the company name and disable edit mode
+    if (accountDetails.profileType === 'company') {
+      registration = registration || (await profile.getProfileProperty('registration'));
+      accountDetails.accountName = registration.company;
+    }
+  } catch (ex) { }
```
Comment (karladler) [inline dapps/ui.libs/src/bccUtils.ts:45]: do nothing?

File: dapps/ui.libs/src/bccUtils.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+ */
+export async function getUserAlias(profile: bcc.Profile, accountDetails?: any, registration?: any) {
+  try {
+    accountDetails = cloneDeep(
+      bcc.lodash,
+      accountDetails || await profile.getProfileProperty('accountDetails') || { }
+    );
+
+    // for companies load directly the company name and disable edit mode
+    if (accountDetails.profileType === 'company') {
```
Comment (karladler) [inline dapps/ui.libs/src/bccUtils.ts:41]: does iot-device needs to be considered?

File: dapps/ui.libs/src/style/forms.scss [hunk 3]
```diff
@@ -47,17 +45,16 @@ $controlSelectors: (
 
   .custom-select {
     -webkit-appearance: none;
```
Comment (karladler) [inline dapps/ui.libs/src/style/forms.scss]: isn't sass handling browser prefixes? So `appearance: none;` should be enough?

File: dapps/ui.libs/src/bccUtils.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      registration = registration || (await profile.getProfileProperty('registration'));
+      accountDetails.accountName = registration.company;
+    }
+  } catch (ex) { }
+
+  // load alias from addressbook, when it's not available
+  if (!accountDetails || !accountDetails.accountName) {
```
Comment (karladler) [inline dapps/ui.libs/src/bccUtils.ts]: early return would be probably better readable:

```suggestion
  if (accountDetails  && accountDetails.accountName) {
    return accountDetails.accountName;
  }
  // load addressbook info ...
```

File: dapps/ui.libs/src/style/forms.scss [hunk 5]
```diff
@@ -47,17 +45,16 @@ $controlSelectors: (
...
     &.form-control {
       border-radius: 4px;
     }
   }
 
   .evan-form {
-
     &.transparent {
-      > div:first-of-type {
+       > div:first-of-type {
```
Comment (karladler) [inline dapps/ui.libs/src/style/forms.scss]: weird indentation ... does ist come from css-comb?

File: dapps/ui.libs/src/bccUtils.ts [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+import * as bcc from '@evan.network/api-blockchain-core';
+import { cloneDeep } from './utils';
+
+/**
+ * Returns the users alias, depending on it's profile type. If it's an old profile, resolve alias
+ * from
```
Comment (karladler) [inline dapps/ui.libs/src/bccUtils.ts]:
```suggestion
 * from address book.
```

File: dapps/ui.libs/src/bccUtils.ts (hunk 2 above)
Comment (Tschuck) [reply dapps/ui.libs/src/bccUtils.ts:41]: for iot devices, the account name should be used

File: dapps/ui.libs/src/bccUtils.ts (hunk 1 above)
Comment (Tschuck) [reply dapps/ui.libs/src/bccUtils.ts:45]: yes, we do not want to log, if an old profile is loaded

**Pull request merged**